src/rendering/scene/Container.ts

```diff
@@ -734,6 +737,8 @@ export class Container<T extends View = View> extends EventEmitter<ContainerEven
         this._skew = null;
 
         this.emit('destroyed');
+
+        this.onDestroyed();
```

Comment:
We call `emit('destroyed')` already
Why do we need the callback?

Reply:
i don't really know! I think i was trying to simplify - but just made it more coplex :D will adjust